pkg/networkservice/common/interpose/server_test.go

```diff
@@ -96,40 +94,25 @@ func TestInterposeServer(t *testing.T) {
 
 	conn, err := client.Request(context.TODO(), request)
 	require.NoError(t, err)
-	require.True(t, touchServer.touched)
+	require.Equal(t, 1, counter.requests)
 
 	// 2. Refresh
 
 	request = request.Clone()
 	request.Connection = conn.Clone()
 
-	touchServer.touched = false
-
-	conn, err = client.Request(context.TODO(), request)
-	require.NoError(t, err)
-	require.True(t, touchServer.touched)
-
-	// 3. Close
-
-	conn = conn.Clone()
-
-	touchServer.touched = false
-
-	_, err = client.Close(context.TODO(), conn)
+	_, err = client.Request(context.TODO(), request)
```

Comment:
Can we change TODO context?

Reply:
Removed commit modifying interpose, guess it is no more part of the PR.